lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -72,18 +74,49 @@ import {
 import { SilentCacheClient } from "./SilentCacheClient";
 import { AuthenticationResult } from "../response/AuthenticationResult";
 import { base64Decode } from "../encode/Base64Decode";
+import { version } from "../packageMetadata";
 
 const BrokerServerParamKeys = {
     BROKER_CLIENT_ID: "brk_client_id",
     BROKER_REDIRECT_URI: "brk_redirect_uri",
 };
 
+/**
+ * Provides MSAL and browser extension SKUs
```

Comment:
Can you explicitly document what you expect to have in skus[0..4].

Reply:
Do we need to document it here in the code given that we have a shared spec?